diff --git a/shell/resources/pps-document-view.blp b/shell/resources/pps-document-view.blp
index 6f27cd96a..316534ad8 100644
--- a/shell/resources/pps-document-view.blp
+++ b/shell/resources/pps-document-view.blp
@@ -195,7 +195,7 @@ template $PpsDocumentView: Adw.BreakpointBin {
 
         [start]
         ToggleButton sidebar_button {
-          icon-name: "sidebar-show-symbolic";
+          icon-name: "view-sidebar-start-symbolic";
           action-name: "doc.show-sidebar";
           tooltip-text: _("Toggle Sidebar");
         }
