diff --git a/shell/resources/pps-document-view.ui b/shell/resources/pps-document-view.ui
index 22a65a1de..079a1a967 100644
--- a/shell/resources/pps-document-view.ui
+++ b/shell/resources/pps-document-view.ui
@@ -204,7 +204,7 @@
                 </child>
                 <child type="start">
                   <object class="GtkToggleButton" id="sidebar_button">
-                    <property name="icon-name">sidebar-show-symbolic</property>
+                    <property name="icon-name">view-sidebar-start-symbolic</property>
                     <property name="action-name">doc.show-sidebar</property>
                     <property name="tooltip-text" translatable="yes">Toggle Sidebar</property>
                   </object>
